Utilites/Tokenizer/tokenize.ps1

```diff
@@ -91,6 +95,12 @@ ForEach($match in $matches)
     catch{
         $variableValue=$match
     }
+
+	Write-Host (Get-LocalizedString -Key 'Token Value: {0}...' -ArgumentList $variableValue) -ForegroundColor Green

```

Comment:
This is probably undesired in the case of "secret" variables? As in - everyone with access to the release will see the value in the log?